chef/cookbooks/updater/recipes/default.rb

```diff
@@ -40,6 +40,31 @@
       zypper_command += " --auto-agree-with-licenses"
     end
 
+    ruby_block "check for HA updates" do
+      block do
+        case node[:updater][:zypper][:method]
+        when "patch"
+          command = "zypper list-patches"
+        when "update"
+          command = "zypper list-updates"
+        when "dist-upgrades"
+          # TODO: check the correct one for here
+          command = "zypper list-updates"
+        end
+        command += '|egrep "corosync|pacemaker" -q'
+        `#{command}`
```

Comment:
I'm not keen on this approach to executing the command.  Backquotes are supposed to be used when the output of the command needs to be captured, but here we are just throwing it away, so `system` would be more appropriate.